### PR TITLE
HTTPC-132: Update HTTP retry mechanism docs

### PR DESCRIPTION
**Pull request by eze210**

### CODE REVIEW

File: http/1.5/modules/ROOT/pages/http-request-ref.adoc [hunk 1]
```diff
@@ -191,7 +191,8 @@ To avoid issues, it's important to consume responses when streaming.
 
 == Retry Mechanisms
 
-To configure how many retries the HTTP Requester can try consuming an external HTTP service, you can either use the Until Successful scope, or the built-in retry mechanism in the requester:
+The reconnection strategy is a mechanism used by the mule SDK to re-establish *Connections* when a *ConnectionException* is caught. To learn more about *Connections*, see xref:mule-sdk::connections.adoc[Connections documentation]. The HTTP Connector understands as a Connection an HTTP Client from the HTTP Service, and not a TCP connection, so it's important to consider that the HTTP Connector will *not* use this mechanism to reconnect sockets. +
+To configure how many *retries* the HTTP Requester can try consuming an external HTTP service, you can either use the Until Successful scope, or the built-in retry mechanism in the requester:
```
Comment (Cristian-Venticinque) [inline http/1.5/modules/ROOT/pages/http-request-ref.adoc]:
```suggestion
The HTTP Connector uses a retry mechanism that enables you to configure how many times it tries consuming an external HTTP Service. The HTTP Connector uses this mechanism to reconnect an HTTP client with an HTTP service. The HTTP connector does not manage TCP connections, so this mechanism does not reconnect sockets.

[WARNING]
The reconnection strategy is a mechanism the mule SDK uses to re-establish _Connections_ when a `ConnectionException` is caught. It does not affect connections to an HTTP Service created by the HTTP Requester.
To learn more about _Connections_, see xref:mule-sdk::connections.adoc[Connections documentation]

To configure how many times the HTTP Requester can try consuming an external HTTP service (_retries_), you can either use the Until Successful scope, or the built-in retry mechanism in the requester:
```

File: http/1.5/modules/ROOT/pages/http-request-ref.adoc [hunk 2]
```diff
@@ -206,7 +207,8 @@ The Until Successful scope processes the components within it, in order, until t
 For additional information on the scope, see xref:mule-runtime::until-successful-scope.adoc[Until Successful Scope].
 
 * Built-in mechanism +
-The HTTP Requester automatically retries by default the idempotent methods `PUT`, `DELETE`, `GET`, `HEAD`, `OPTIONS`, and `TRACE`, three times before it fails. Thereby, you can configure two system properties for this mechanism:
+The HTTP Requester automatically retries by default the idempotent methods `PUT`, `DELETE`, `GET`, `HEAD`, `OPTIONS`, and `TRACE`, three times before it fails, when it receives an error of type *'Remotely Closed'*. +
+Thereby, you can configure two system properties for this mechanism:
```
Comment (Cristian-Venticinque) [inline http/1.5/modules/ROOT/pages/http-request-ref.adoc]:
```suggestion
When the HTTP Requester receives an error of type `Remotely Closed`, the requester automatically retries by default the idempotent methods `PUT`, `DELETE`, `GET`, `HEAD`, `OPTIONS`, and `TRACE`, three times before it fails. +
Thereby, you can configure two system properties for this mechanism:
```

**Review by Cristian-Venticinque — COMMENTED** [review]
Please check suggestions.

File: http/1.5/modules/ROOT/pages/http-request-ref.adoc [hunk 3]
```diff
@@ -191,7 +191,13 @@ To avoid issues, it's important to consume responses when streaming.
 
 == Retry Mechanisms
 
-To configure how many retries the HTTP Requester can try consuming an external HTTP service, you can either use the Until Successful scope, or the built-in retry mechanism in the requester:
+The HTTP Connector uses a retry mechanism that enables you to configure how many times it tries consuming an external HTTP Service. The HTTP Connector uses this mechanism to reconnect an HTTP client with an HTTP service. The HTTP connector does not manage TCP connections, so this mechanism does not reconnect sockets.
+
+[WARNING]
+The reconnection strategy is a mechanism the mule SDK uses to re-establish _Connections_ when a `ConnectionException` is caught. It does not affect connections to an HTTP Service created by the HTTP Requester.
+To learn more about _Connections_, see xref:mule-sdk::connections.adoc[Connections documentation]
```
Comment (kevintroller) [inline http/1.5/modules/ROOT/pages/http-request-ref.adoc]:
```suggestion
The HTTP Connector uses a retry mechanism that enables you to configure how many times it tries consuming an external HTTP service. The HTTP Connector uses this mechanism to reconnect an HTTP client with an HTTP service. The HTTP Connector does not manage TCP connections, so this mechanism does not reconnect sockets.

[WARNING]
The reconnection strategy is a mechanism the Mule SDK uses to re-establish _Connections_ when a `ConnectionException` is caught. It does not affect connections to an HTTP service created by the HTTP Requester.
To learn more about _Connections_, see xref:mule-sdk::connections.adoc[Connections documentation]
```

**Review by kevintroller — DISMISSED** [review]

**Review by fermujica — APPROVED** [review]

**Pull request merged**